Attribute_tuning.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+import pandas as pd
+import logging
+from sklearn.metrics.pairwise import cosine_similarity
+
+logging.basicConfig(level = logging.DEBUG)
+
+user_cols = ['user_id','age','gender','occupation','zip_code']
+#importing the data files onto dataframes
+df_users = pd.read_csv('u.user', sep='|', names=user_cols, encoding='latin-1')
+
+def nearest_5years(x, base=5):
```

Comment:
you could generalize this to: `def nearest_x_years` and allow user to pass in a param